### PR TITLE
Update pr-review.md

### PR DESCRIPTION
**Pull request by dvhirst**
Reading for content; editing for consistency. 
Note:  Check for Drupal dependencies in the DB upgrade case.

### CODE REVIEW

File: docs/core/pr-review.md [hunk 1]
```diff
@@ -87,8 +83,8 @@ An easy way to do this is:
 
 1. Install a buildkit site for the latest publicly available release of CiviCRM (*not* `master`). Pass the `--civi-ver` option to civibuild for this.
 1. Update the `civicrm` directory files so that the codebase has the changes in the PR (perhaps by using [Hub](https://hub.github.com/) as described above).
-1. From the `civicrm` directory, run `./bin/setup.sh -Dg` to update the generated-code
-1. Run `drush civicrm-upgrade-db` to perform database upgrades
+1. From the `civicrm` directory, run `./bin/setup.sh -Dg` to update the generated-code.
+1. Run `drush civicrm-upgrade-db` to perform database upgrades. *(TODO:  Check for Drupal dependencies.  If Wordpress or Joomla! is the target CMS, how should database upgrades be accomplished?)*
```
Comment (seanmadsen) [inline docs/core/pr-review.md]: @dvhirst it seems like your "TODO" comment here is intended to be a to-do item for docs editors (not readers). Am I correct? If so, I'd rather we leave this content out of the guide and instead open a ticket within this repo to track this improvement. This way readers will have a more polished experience and editors will be more likely to spot (and fix) the issue. Is that okay? If you're cool with this approach, then I think it'll merge this PR after you remove this "TODO" bit.

Comment (dvhirst) [reply docs/core/pr-review.md]: I'm OK with your approach, and will remove the TODO item.  You'd asked me to highlight Drupal dependencies as I found them, this was the initial one I spotted.  In future, I'll mark them with a new issue ticket in the appropriate docs folder.

**Comment by dvhirst** [comment]
Editorial TODO removed...

**Comment by seanmadsen** [comment]
Awesome. Thanks Don.

**Pull request merged**